Library/Formula/libantlr3c.rb

```diff
@@ -1,9 +1,12 @@
-require 'formula'
-
 class Libantlr3c < Formula
-  homepage 'http://www.antlr3.org'
-  url 'http://www.antlr3.org/download/C/libantlr3c-3.4.tar.gz'
-  sha1 'faa9ab43ab4d3774f015471c3f011cc247df6a18'
+  homepage "http://www.antlr3.org"
+  url "http://www.antlr3.org/download/C/libantlr3c-3.4.tar.gz"
+  sha1 "faa9ab43ab4d3774f015471c3f011cc247df6a18"
+  revision 1
+
+  option "with-exceptions", "Compile with support for exception handling"
+  option "with-antlrdebug", "Compile with antlr debugger in runtime"
```

Comment:
We need the exceptions flag in order for the library to work properly with clients that use C++-style exceptions.

If we just always turned it on, my guess is that it would cause a slight performance hit for clients that don't need the extra stack unwinding information. I doubt this would be a huge deal, but I'll trust your judgment (I am very green when it comes to writing formulas; I really appreciate your help)